data/xsum/data.txt

```diff
@@ -0,0 +1,24 @@
+
```

Comment:
I think this was just scratch work, should this be deleted?

Reply:
Yes, good catch